cat-theme/src/css/header/_navigation.scss

```diff
@@ -0,0 +1,233 @@
+#navigation {
+    align-items: center;
+
+    @include flex(center, flex-start, space-between);
+
+    position: fixed;
+    width: 100%;
+    z-index: 5000;
```

Comment:
Does this `z-index` really need to be set to `5000`?  For `z-index`, we should try to align with the standards we have set in https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js#L1943...L1959.